src/interfaces/Inspection.ts

```diff
@@ -141,6 +146,13 @@ export const createDefaultInspection = (): Inspection => ({
     fr: [createDefaultSpecification()],
   },
   first_aid: { en: [], fr: [] },
-  guaranteed_analysis: [createDefaultNamedValue()],
+  guaranteed_analysis_fr: {
```

Comment:
Tested with:

![granulaine](https://github.com/user-attachments/assets/fe37329b-9f9c-478a-a223-9eef9990e2b3)

Received this from `post /inspections`:
```json
{
	"guaranteed_analysis": {
		"en": [
			{
				"edited": false,
				"name": "Total nitrogen (N)",
				"unit": "%",
				"value": 10
			},
			{
				"edited": false,
				"name": "Soluble potash (K2O)",
				"unit": "%",
				"value": 4
			},
			{
				"edited": false,
				"name": "Organic matter",
				"unit": "%",
				"value": 61.5
			},
			{
				"edited": false,
				"name": "Maximum moisture content",
				"unit": "%",
				"value": 14.8
			}
		],
		"fr": [
			{
				"edited": false,
				"name": "Azote totale",
				"unit": "%",
				"value": 10
			},
			{
				"edited": false,
				"name": "Potasse soluble",
				"unit": "%",
				"value": 4
			},
			{
				"edited": false,
				"name": "Matière organique",
				"unit": "%",
				"value": 61.5
			},
			{
				"edited": false,
				"name": "Teneur maximum en humidité",
				"unit": "%",
				"value": 14.8
			}
		],
		"is_minimal": null,
		"title": "Guaranteed minimum analysis",
		"titre": "Analyse minimum garantie"
	}
}
```

Page showed:
![image](https://github.com/user-attachments/assets/4d6e9879-70b3-438e-9b0c-00d588931484)